ignite/pkg/protoanalysis/protoutil/creator_test.go

```diff
@@ -435,7 +436,7 @@ func TestCreateEnum(t *testing.T) {
 	}
 }
 
-// OneOf fields:
+// OneOf fields:.
```

Comment:
I think we can remove these comments

Reply:
I agree.  There's a linter causing it actually.  I'll get in there on that.